test/PrintAsObjC/textual-imports.swift

```diff
@@ -0,0 +1,36 @@
+// REQUIRES: objc_interop
+
+// RUN: %empty-directory(%t)
+// RUN: %target-swift-frontend(mock-sdk: -sdk %S/../Inputs/clang-importer-sdk -I %t) -typecheck -emit-objc-header-path %t/textual-imports.h -emit-objc-header-textually %s 
+// RUN: %FileCheck %s < %t/textual-imports.h
+// RUN: %check-in-clang -fno-modules -Qunused-arguments %t/textual-imports.h
+
+import Foundation
+
+public class HelloWorld: NSObject {
+  @objc public func sayHello() {
+    print("Hello, World!")
+  }
+
+  @objc public func getPoint() -> CGPoint {
+    return CGPoint(x: 1, y: 1)
+  }
+}
+
+// CHECK:      #if __has_feature(modules)
+// CHECK-NEXT: #if __has_warning("-Watimport-in-framework-header")
+// CHECK-NEXT: #pragma clang diagnostic ignored "-Watimport-in-framework-header"
+// CHECK-NEXT: #endif
+// CHECK-NEXT: @import CoreGraphics
+// CHECK-NEXT: @import ObjectiveC;
+// CHECK-NEXT: #else
+// CHECK-NEXT: #if __has_include(<include/CoreGraphics.h>)
+// CHECK-NEXT: #import <include/CoreGraphics.h>
```

Comment:
This import actually fails, but the guard around it prevents the test from revealing it.